src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/entities/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.training.designpatterns.fastfoodrestaurant.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Condiment;
+import com.epam.training.designpatterns.fastfoodrestaurant.food.Food;
+import com.epam.training.designpatterns.fastfoodrestaurant.strategies.CookingStrategy;
+import com.epam.training.designpatterns.fastfoodrestaurant.strategies.DefaultCookingStrategy;
+
+
+public class Order {
+	
+	private Client client;
+	private Food food;
```

Comment:
A rendelésbe már egy `Food` példány kerül be, ami még nincs kész, mert a szakács még dolgozni fog rajta. Erre többféle megoldást láttam a PR-ekben és felmerül a közös vagy külön (nem triviális) domain problémája. Erről írni fogok majd a csoportnak külön.

Reply:
Igen-igen, ez bennem is felmerült, és kicsit szerencsétlen az elnevezés, mert az Order-ben lévő Food mező az egy még nem elkészült, rendelés alatt álló 'ételt' jelent, míg a ReadyMeal class-ban lévő Food pedig már egy elkészült ételt. 
Ehhez hasonló elő szokott fordulni valós projekteken, hogy egy class mást reprezentál, attól függően hogy mi a kontextus/pl. melyik osztály tartalmazottjaként jelenik meg éppen?

Reply:
Talán egy megoldás lehetne az is, hogy az Order-be csak a Food mint class kerül be (pl Chips.class), és majd a Chef példányosítja ez alapján

Reply:
Igen, jó kérdések, és válaszoltam rájuk az emailben, illetve ott vannak a példabranch-ek az eredeti repóban.